### ISSUE TITLE
Find P-Value in subsystems/Drive.java in order for the robot to drive straight with Gyro

### ISSUE THREAD

**Issue by kushagharahi** [PearceRobotics/rapid-react-2022#8]
- Create a branch called drive-straight
- Make sure you enable gyro via setting `gyroDisabled` boolean to false in `subsystems/Drive.java`
- Commit changes to branch you created above
- Open Pull Request against the main branch to merge in your code

This ticket depends on #3 being done and merged into the main branch
